schema/spacewalk/upgrade/susemanager-schema-4.4.4-to-susemanager-schema-4.4.5/012-update-queue_server.sql

```diff
@@ -0,0 +1,54 @@
+--
+-- Copyright (c) 2008--2015 Red Hat, Inc.
+--
+-- This software is licensed to you under the GNU General Public License,
+-- version 2 (GPLv2). There is NO WARRANTY for this software, express or
+-- implied, including the implied warranties of MERCHANTABILITY or FITNESS
+-- FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv2
+-- along with this software; if not, see
+-- http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+-- 
+-- Red Hat trademarks are not licensed under GPLv2. No permission is
+-- granted to use or replicate Red Hat trademarks that are incorporated
+-- in this software or its documentation. 
+--
+--
+--
+--
+
+-- the next two views are basically the same.  the first, though, has an outer join to
+-- the errata stuff, in case there are packages the server needs that haven't been
+-- errata'd (ie, the fringe case)
+
+
+CREATE OR REPLACE FUNCTION
+queue_server(server_id_in IN NUMERIC, immediate_in IN NUMERIC DEFAULT 1)
```

Comment:
Same for this one

Reply:
It is used: in rhn_channel.update_needed_cache(), rhn_channel.subscribe_server() and  rhn_channel.unsubscribe_server()